samples/core/runtime_resource_requests/resource_request.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import kfp
+from kfp import dsl, components
+
+@components.create_component_from_func
+def print_op():
+   '''Print hello world'''
```

Comment:
nit: always use """triple double quotes""" around docstrings. and the summary line should be terminated by a period, question mark, or exclamation point.
https://google.github.io/styleguide/pyguide.html#381-docstrings

Reply:
Thanks, will read through the style guide.